demos/setup-project.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Can we callout this file in the README.md file for the demos directory?

Reply:
Yes, will do